TestControllers/AzureController.psm1

```diff
@@ -83,15 +76,21 @@ Class AzureController : TestController
 			}
 		}
 		elseif ($this.ARMImageName) {
-			&$ValidateARMImageName
+			$ArmImagesToBeUsed = @($this.ARMImageName.Trim(", ").Split(',').Trim())
+			if ($ArmImagesToBeUsed | Where-Object {$_.Split(" ").Count -ne 4}) {
+				$parameterErrors += ("Invalid value for the provided ARMImageName parameter: <'$($this.ARMImageName)'>." + `
+									 "The ARM image should be in the format: '<Publisher> <Offer> <Sku> <Version>,<Publisher> <Offer> <Sku> <Version>,...'")
```

Comment:
current test summary like this. can we add kernel version into the test set up or below it? if it is big change, maybe postpone it 

![image](https://user-images.githubusercontent.com/10083705/88908397-63369f80-d28c-11ea-85ed-93dba549ff46.png)

Reply:
yes, updated. added Kernel info, see TestLogs.  if the Kernel has been changed, it will display  Kernel Version  xxx -> yyy